source/mainnet/net/resources/ccd-scan-staking.rst

```diff
@@ -0,0 +1,118 @@
+
+|
+
+- **Account address**: the first six characters of the account hash. You can click Copy |copy| to copy the entire hash. If you click on an account hash, the :ref:`account details<accounts-view>` appears.
+- **Delegation target**: the baker whose pool the account has staked. If you click on a baker ID, the :ref:`baker details<home-screen-baker>` appears.
+- **Restake earnings**: shows the delegator's restaking preference.
+- **Staked amount**: amount of CCD the delegator has staked.
+
+Passive delegation view
+=======================
+
+If you select **Passive delegation**  it is possible to see the size of the passive delegation in total and information about the yield from delegating to passive delegation. Furthermore it is possible to see the commission rates for passive delegation.
+
+.. image:: ../images/ccd-scan/ccd-scan-passive-delegation.png
+
+- **Delegated stake**: amount of CCD delegated to passive delegation.
+- **APY (7 days)**: the calculated yield of passive delegation based on past 7 days.
+- **APY (30 days)**: the calculated yield of passive delegation based on past 30 days.
+- **Baking commission**: percent of rewards paid as commission to baker.
+- **Finalisation commission**: percent of finalization commission.
+- **Transaction commission**: the transaction commission percent.
+- **Rewards**: list of rewards paid out during the selected time range. Includes an overview graph that shows the aggregated rewards for passive delegation based on the applied filter. Rewards details are displayed in a table.
+    - **Time** shows the date/time the reward was paid
+    - **Reference** shows the first six characters of the block hash in which the reward was paid. You can click Copy |copy| to copy the entire hash. Click the hash to see the :ref:`block details<home-screen-block>` for the block where the award payout occurred.
+    - **Rewards** shows the amount of reward distributed.
+- **Delegators**: the delegators staking to passive delegation, their restake preference, and amount staked. Click on a **Hash** to see :ref:`account details<accounts-view>`. You can click Copy |copy| to copy the entire hash.
+
+.. _home-screen-baker:
+
+Baker details
+=============
+
+When you click a baker ID, the following appears:
+
+.. image:: ../images/ccd-scan/ccd-scan-baker-details.png
+
+|
+
+The baker details shows the following information about the baker.
+
+- **Baker**: the baker ID. This is the ID you need if you want to delegate to the baker pool in a wallet. If the Baker has been removed (i.e. the baker is no longer active), **Removed** appears next to the ID and a timestamp shows when the baker stopped.
+- If there are any pending changes, they are shown here. Events such as stake increase, stake reduction, or baker stop are displayed.
+- **Total stake**: the baker's total stake. Below the total is the amount staked by the baker and the amount of stake that comes from delegators.
+- **Open, Closed**: the baker pool status.
```

Comment:
Perhaps elaborate on this. Open means you can delegate to it, closed means you cannot.